src/main/java/com/ibm/devops/connect/CRPipeline/UploadASoCTestResult.java

```diff
@@ -109,6 +113,30 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         String buildUrlValue = envVars.expand(this.buildUrl);
         String metricDefinitionValue = envVars.expand(this.metricDefinition);
         String recordNameValue = envVars.expand(this.recordName);
+        String instanceBaseUrl = envVars.expand(this.instanceBaseUrl == null ? "" : this.instanceBaseUrl.toString());
+
+        List<Entry> entries = Jenkins.getInstance().getDescriptorByType(DevOpsGlobalConfiguration.class).getEntries();
+        int instanceNum = 0;
+        if(instanceBaseUrl.equals("Upload ASoC Test Result to All UCV Instances")){
```

Comment:
How is the user going to know what the value of this string is?  Is it the default?  It might make more sense for it to be optional since it seems that the `else` case does the same thing.